src/components/DeleteModalContainer.tsx

```diff
@@ -0,0 +1,50 @@
+import { Dispatch, RefObject, SetStateAction } from 'react';
+import { Task } from '../utils/types';
+import { deleteAllTasks } from '../utils/api';
+import DeleteModalUI from './DeleteModalUI';
+
+interface DeleteModalContainerProps {
+  deleteModalRef: RefObject<HTMLDialogElement>;
```

Comment:
Fine to use a ref for now; we'll talk about how to make this better and less ref dependent in the future.

Reply:
made a note in notion to chat about